src/components/atoms/authors/authors.module.scss

```diff
@@ -18,19 +19,94 @@
 
   display: inline;
 
-  &::after {
+  &::before {
     content: ", ";
   }
 
-  &:last-child::after {
-    content: "";
+  &:first-of-type {
+    &::before {
+      content: "";
+    }
+  }
+
+  .authors--collapsible & {
+    &:nth-of-type(1n+3) {
+      display: none;
+    }
+
+    &:last-of-type {
+      display: inline;
+
+      &::before {
+        content: " ... ";
+      }
+    }
+
+    @media only all and (min-width: settings.$breakpoint-medium) {
+      &:nth-of-type(1n+3) {
+        display: inline;
+      }
+
+      &:nth-of-type(1n+10) {
+        display: none;
+      }
+    }
+
+    &:last-of-type { /* stylelint-disable-line no-duplicate-selectors */
+      display: inline;
+    }
+  }
+
+  .authors-list--expanded & { /* stylelint-disable-line no-descending-specificity */
```

Comment:
@discodavey I would appreciate your input to see if these lint disable can be avoided. I don't want to assume that I can't refactor this and conform with the linting and have the intended effect.

Reply:
What is the linting error?

Reply:
> What is the linting error?

`no-descending-specificity`